discourse_migration/userguide.md

```diff
@@ -45,48 +45,49 @@ are the typical use cases:
 
 This table explains the mapping from mailing lists to categories in Discourse.
 The email addresses of these categories will remain the same, after the
+migration.  Deprecated lists will become read-only as part of the Discourse
 migration.
 
 | Mailing lists         | category in Discourse |
 |-----------------------|--------------------|
 | All-commits           | no migration at the moment |
 | Bugs-admin            | no migration at the moment |
 | cfe-commits           | no migration at the moment |
-| cfe-dev               | llvm-project/clang |
-| cfe-users             | **TODO** llvm-project/clang or create new category? |
-| clangd-dev            | llvm-project/clangd |
-| devmtg-organizers     | **TODO** create new category |
+| cfe-dev               | Clang Compiler & Tools |
+| cfe-users             | Clang Compiler & Tools/Using Clang |
+| clangd-dev            | Clang Compiler & Tools/clangd |
+| devmtg-organizers     | Deprecated |
 | Docs                  | infrastructure/documentation |
-| eurollvm-organizers   | **TODO** create new category |
+| eurollvm-organizers   | Deprecated |
 | flang-commits         | no migration at the moment |
-| flang-dev             | llvm-project/flang |
+| flang-dev             | LLVM Based Subprojects/flang |
 | gsoc                  | community/gsoc |
 | libc-commits          | no migration at the moment |
-| libc-dev              | llvm-project/libc |
-| Libclc-dev            | **TODO** create new category |
+| libc-dev              | Runtimes/libc |
+| Libclc-dev            | Runtimes/libclc |
```

Comment:
What do you think of the alternative:

`Runtimes/C - libc`
`Runtimes/C++ - libcxx`
`Runtimes/OpenCL - libclc`

etc?  That will be more discoverable/understandable for new people.

Reply:
I agree on this. Much easier to understand.

Reply:
> What do you think of the alternative:
> 
> `Runtimes/C - libc`
> `Runtimes/C++ - libcxx`
> `Runtimes/OpenCL - libclc`
> 
> etc? That will be more discoverable/understandable for new people.
 
So your suggestion is to use "Runtimes/C - libc" as the full category name with no sub-categories?

Also, what about OpenMP and sanitizers.  We had those as sub-categories in the Runtime category.  Should those also become top-level categories?

Reply:
I read it as 
Runtimes/C
Runtimes/C++
Runtimes/OpenCL 

And then the - name was just showing what it mapped to. Maybe I misunderstood.

Reply:
I meant a top level "Runtimes" category and then "C - libc" as a subcategory.  I think that putting "OpenMP" and "Sanitizers" as subcategory of Runtimes would also make a lot of sense.

My understanding is that IWG is focused on getting the mailing lists mapped out.  Once the base structure is in place (e.g. the top level Runtimes category) we can fill in the new entries on demand.

Reply:
Incidentally, I think it would also be worth going simpler than this.  The name of the runtime library is confusing in many cases.  I think it is worth considering just doing:

```
Runtimes / C
Runtimes / C++
Runtimes / OpenCL
Runtimes / OpenMP
Runtimes / Sanitizers
Runtimes / Other
```

Reply:
Could we have the top-level 'Runtimes' category be for posts about other runtimes, or do you think it's better to have a separate sub-category?

Reply:
Both work.  The downside of reusing the top level for this is that you can't follow "just that".  You'd have to follow the entire "Runtimes" category and you'd get email about everything.  This would be annoying if you just care about polly or something, you'd rather just follow "Runtimes/Other".

Reply:
btw, it isn't clear to me that even "Sanitizers" will be high volume.  I think we should either start with a minimal set of sub categories (just to make the mailing list transition work) and add on demand, or go full in and add subcategories for all the things we know of.